Updates for use with Gazebo 9

ogallagher opened the pull request:
I was able to run the most basic ROS control example with Gazebo 9 after installing some new dependencies and modifying a couple of source files. I described the changes in the README.

ogallagher committed:
Created README.md

Still working on explaining code changes, and my modified pasted of the Google Doc is probably not 100% correct, but I’m not motivated to double-check.

ogallagher committed:
Linked ROS packages

In src/gazebo_bridge.cc the #include statements were returning file-not-found errors.

ogallagher committed:
Swapped use of gazebo::math with ignition::math

ogallagher committed:
Mentioned source changes

ogallagher committed:
Update README.md

ogallagher committed:
Added ignition libraries install step

dawonn commented:
Thank you for the Pull Request! Please update the readme for inclusion her tho, looks like there are personal comments that mat not make sense once merged. I'm very glad you moved to the Google Doc content into a readme, should have happened long ago.... ;)

ogallagher commented:
Alright! Thanks for looking into it; soon I plan to confirm what I added and I’ll take out the personal comments to make it merge-able

ogallagher committed:
Fixed README

Formatted, removed merge-request comments, modified some directions.

ogallagher committed:
Expanded usage explanation

ogallagher committed:
Fixed syntax detail

ogallagher commented:
I modified README.md for the merge, but I’d like to have the rest of my explanation of the Gazebo demo filled out, explaining what urdf.launch and cmd_vel_mux.launch do. Could @dawonn or someone else add that part in?

ogallagher committed:
Fixed explanation of model modification

The hokuyo laser has to be added to the file; it's not included in the version that is downloaded from the Gazebo model database.
